addons/web_editor/models/ir_attachment.py

```diff
@@ -25,6 +25,7 @@ class IrAttachment(models.Model):
     image_width = fields.Integer(compute='_compute_image_size')
     image_height = fields.Integer(compute='_compute_image_size')
     original_id = fields.Many2one('ir.attachment', string="Original (unoptimized, unresized) attachment")
+    is_favorite = fields.Boolean()
```

Comment:
Maybe a better name should be found, let's not overthink it for now while we got an approval from the framework-py team but something in the name that would hints that it is related to:
1. Images attachment
2. Website builder / Web Editor

Would surely be great.

Reply:
The `ir_attachment` table holds too many records to be `ALTER TABLE` in a timely manner, you can add non-stored fields or fields stored in another table, but anything that would require an `ALTER TABLE` on this table is a big no-no

Reply:
Moreover, 
> users to mark attachments as favorite 

Sound like it's a per-user choice, so a m2m between ir.attachment and res.users

Reply:
Ty :)

The goal was to not have the feature per user but shared.
So you share your main images for everyone being part of the website creation.

But we will actually give it another round of discussion / specification to be sure.
`favorite_user_ids` is the usual way of doing the "per user favorites", we will see if we can go with that "per user" spec.

Reply:
Could be but last specs is "not a user choice"... now it does not mean it will stay that way in a year so... :grimacing: We are discussing it.